.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9, 3.10.8, 3.11]
```

Comment:
Could we just as easily add 3.12 to the mix too? Perhaps no further changes will be needed other than this line.